web/vite.config.ts

```diff
@@ -0,0 +1,68 @@
+import path from 'node:path'
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import { resolve } from 'path'
+import svgr from 'vite-plugin-svgr'
+import istanbul from 'vite-plugin-istanbul'
+
+export default defineConfig({
+  build: {
+    sourcemap: true,
+    outDir: 'build'
+  },
+  optimizeDeps: {
+    include: ['@mui/material/Tooltip', '@emotion/styled/base', '@mui/material/Unstable_Grid2']
```

Comment:
Were these added to avoid the vite build errors you mentioned earlier?

Reply:
yep

Reply:
Ok cool, is it possible to add a comment in this config ts file explaining that a little bit with links to the issue?

Reply:
done